v03_pipeline/lib/tasks/update_project_table_with_deleted_families.py

```diff
@@ -0,0 +1,31 @@
+import hail as hl
+import luigi
+
+from v03_pipeline.lib.misc.family_entries import remove_family_guids
+from v03_pipeline.lib.tasks.base.base_project_table_task import BaseProjectTableTask
+
+
+class UpdateProjectTableWithDeletedFamiliesTask(BaseProjectTableTask):
+    family_guids = luigi.ListParameter()
+
+    def complete(self) -> bool:
+        return super().complete() and hl.eval(
+            hl.bind(
+                lambda family_guids: (
+                    hl.all(
```

Comment:
why don't we also need a line like [this](https://github.com/broadinstitute/seqr-loading-pipelines/pull/761/files#diff-ba28fb0f9bee045c745fd70e50d7ec743dff2b98cc03b7299469dc00501e294bR30) here?

Reply:
ah!  good question.  that line is handling the case where the project is missing from the lookup table (which contains all of our projects!).  Within the project table, we don't have that notion.  If a project is missing, the table itself is missing (which would be handled by the `super().complete()`!